python/tvm/relax/frontend/torch/fx_translator.py

```diff
@@ -740,7 +760,9 @@ def from_fx(self, model, input_info: List[Tuple[Tuple[int], str]]) -> tvm.IRModu
         return self.block_builder.get()
 
 
-def from_fx(model, input_info: List[Tuple[Tuple[int], str]]) -> tvm.IRModule:
+def from_fx(
+    model, input_info: List[Tuple[Tuple[int], str]], keep_params_as_input: bool = False
+) -> tvm.IRModule:
     """Convert a PyTorch FX GraphModule to a Relax program
```

Comment:
@vinx13 can you upstream this part of the change to unity? to reduce set of merge errors